main.go

```diff
@@ -110,7 +110,68 @@ func layout(g *gocui.Gui) error {
 	}
 	return nil
 }
+func scrollUP(v *gocui.View) error {
+	scrollView(v, -1)
+	return nil
+}
+func scrollDOWN(v *gocui.View) error {
```

Comment:
Here as well

Reply:
resolved in commit e4784e4fb27aed0e0386db1e80b4613609645775